### ISSUE TITLE
Considering deeplabv3 model?

### ISSUE THREAD

**Issue by John1231983** [GeorgeSeif/Semantic-Segmentation-Suite#4]
Thank you so much for the great project. 
I am reading deeplabv3 and could you consider to implement the deeplabv3 (https://arxiv.org/abs/1706.05587)? It achieved very high performance.   
It also used pre-train resnet 101 model and train the BatchNorm layer. 
Thanks

**Comment by GeorgeSeif** [comment]
Sure I'll take a look at implementing it!

**Comment by John1231983** [comment]
Great. I will wait your performance. Currently, my implementation got around 74% in batch size of 8 after 120k iters, comparison with 75.85% as the paper. Hope you can achive near the paper result

**Comment by GeorgeSeif** [comment]
On which data set is that accuracy? I only saw PASCAL VOC and Cityscapes in the paper.

**Comment by John1231983** [comment]
The pascal voc that trained on aug dataset

**Comment by GeorgeSeif** [comment]
I'll give it a go once I have a chance!
